### ISSUE TITLE
SmartCity-Accessibility #164 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#115]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
[...truncated 130 lines...]

Running smartcity.accessibility.socialnetwork.BrokenAdminTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.251 sec - in smartcity.accessibility.socialnetwork.BrokenAdminTest
Running smartcity.accessibility.socialnetwork.MockAdminTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 4.798 sec - in smartcity.accessibility.socialnetwork.MockAdminTest
Running smartcity.accessibility.mapmanagement.LocationTest
-- Product name: JxMaps
-- Licensed version: 1.x
-- Licensed to: 
-- License type: Evaluation
-- Generation date: Dec 7, 2016
-- Expiration date: Aug 25, 2017
-- License info: Single-user license
-- Current date: Dec 29, 2016
JxMaps license valid.
Yahalom St 21, Modi'in-Maccabim-Re'ut, Israel
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 3.641 sec - in smartcity.accessibility.mapmanagement.LocationTest
Running smartcity.accessibility.search.SearchTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 0.032 sec - in smartcity.accessibility.search.SearchTest
Running smartcity.accessibility.navigation.NavigationTest
88157718
שדרות תל חי
Tests run: 5, Failures: 0, Errors: 0, Skipped: 2, Time elapsed: 9.34 sec - in smartcity.accessibility.navigation.NavigationTest

Results :

Failed tests: 
  BestReviewsTest.getMostRatedTest:52 expected:<smartcity.accessibility.socialnetwork.Review@3a4a5f3c> but was:<smartcity.accessibility.socialnetwork.Review@4f3356c0>

Tests run: 42, Failures: 1, Errors: 0, Skipped: 3

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 51.845 s
[INFO] Finished at: 2016-12-29T16:20:41+00:00
[INFO] Final Memory: 28M/200M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.17:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/workspace/SmartCity-Accessibility/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:
- [KoralChapnik] 99dcf10f5cfd202cd1741bd4ce577ea2b649a922 - changed score max value to be 5 #97

[View full output](http://163.172.171.65:8080/job/SmartCity-Accessibility/164/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
